libs/service-portal/core/src/lib/messages.ts

```diff
@@ -535,6 +535,26 @@ export const m = defineMessages({
     id: 'service.portal:accessControl',
     defaultMessage: 'Aðgangsstýring',
   },
+  accessControlV2: {
+    id: 'service.portal:accessControlV2',
+    defaultMessage: 'Aðgangsstýring V2',
+  },
+  accessControlDelegations: {
```

Comment:
These should not be in service portal core messages. Instead use a seperate file for translation strings with the namespace: **sp.access-control-delegations**. 
If there are any other namespaces inside core the extract string command will delete other 😞

Reply:
I see the comment came out pretty confusing. I was talking about what a few strings beneath the navigation string.

Reply:
Ahhh yes I misunderstood. Thx I will move it :)

Reply:
Or on second though as I look at it better. I see that one of the string is unused so I removed it. However the other three are all used in masterNavigation, i.e.

```
// Aðgangsstýring umboð
      {
        name: m.accessControlV2, // Here
        path: ServicePortalPath.AccessControlDelegations,
        icon: {
          icon: 'lockClosed',
        },
        description: m.accessControlDescription,
        children: [
          {
            name: m.accessControlDelegations, // Here
            path: ServicePortalPath.AccessControlDelegations,
            children: [
              {
                name: m.accessControlGrant, // Here
                path: ServicePortalPath.AccessControlDelegationsGrant,
                navHide: true,
              },
            ],
          },
          {
            name: m.accessControlDelegationsToMe, // Here
            path: ServicePortalPath.AccessControlDelegationsToMe,
            navHide: true,
          },
          {
            name: m.accessControlAccess, // here
            path: ServicePortalPath.AccessControlDelegationAccess,
            navHide: true,
          },
        ],
      },
```

Reply:
Okey great, just make sure they have the same namespace as other core messages 👍

Reply:
Ahh yes thx 🤦